test/e2e/specs/treeview/00-static-treeview.spec.js

```diff
@@ -37,23 +37,132 @@ exports.tests = function (appName, baseUrl) {
+        expect(element.all(by.css('.jstree-node .jstree-search')).get(0).getCssValue('font-style')).toBe('italic');
+      });
+
+
+      it('search `mouse (EMAPA:25765)` takes the scroll to it and is in italics and highlighted', function() {
+        searchInput.clear();
+        searchInput.sendKeys('mouse (EMAPA:25765)');
+        var ele = element.all(by.css('.jstree-node .jstree-search')).get(0);
+        browser.wait(EC.presenceOf(ele), browser.params.defaultTimeout);
+
+        // expect(deriva.elementInViewport(ele)).toBeTruthy('searched node of the tree is not visible.');
+        expect(ele.getCssValue('background-color')).toBe('rgba(239, 239, 166, 1)');
+        expect(ele.getCssValue('font-style')).toBe('italic');
+      });
+
+      it('filter terms for a specific Theiler Stage: Refereshes on stage change', function(done) {
+        browser.wait(EC.elementToBeClickable(filterDropDown), browser.params.defaultTimeout);
+        filterDropDown.click();
+        element(by.id('number-menu')).all(by.tagName('li')).each(function(ele) {
+          browser.wait(EC.elementToBeClickable(ele), browser.params.defaultTimeout);
+          ele.getText().then(function(text) {
+             expect(expectedValue.dropdown).toContain(text, 'dropdown does not contain text.');
+           });
+           done();
+        }).catch(function (err) {
+            done.fail();
+            console.log(err);
+        });
+
+        element(by.id('ui-id-30')).click().then(function() {
+          return deriva.waitForElementInverse(element.all(by.id("loadIcon")).get(0));
+        }).then(function() {
+            expect(element(by.id('jstree')).isDisplayed()).toBeTruthy('Treeview is not visible.');
+            expect(element(by.id('error-message')).isDisplayed()).toBe(false);
+            done();
+        }).catch(function (err) {
+            done.fail();
+            console.log(err);
+        });
+      });
+
+      describe('treeview', function() {
 
         it('it is displayed with no errors', function() {
           expect(element(by.id('jstree')).isDisplayed()).toBeTruthy('Treeview is not visible.');
           expect(element(by.id('error-message')).isDisplayed()).toBe(false);
         });
+
+        describe('hierarchy', function() {
+          beforeAll(function() {
+            btn = element(by.css('#nid_9112 > i'));
+            deriva.waitForElement(btn);
+          });
+
+          it('node list expanded on + click', function() {
+            btn.click();
+            browser.wait(EC.visibilityOf(element(by.css('#nid_9112 > ul'))), browser.params.defaultTimeout);
+            element(by.css('#nid_9112')).all(by.tagName('li')).each(function (elm) {
+              expect((elm).isDisplayed()).toBeTruthy('Expanded List list is not visible.');
+            });
+          });
+
+          it('node list collapse on - click', function() {
+            btn.click();
```

Comment:
This click event is sometimes triggering for me then failing with this message:
`element click intercepted: Element <i class="jstree-icon jstree-ocl" role="presentation"></i> is not clickable at point (33, 1238). Other element would receive the click: <i class="jstree-icon jstree-ocl" role="presentation"></i>`

I'm not sure what is wrong with this test case without picking it apart and really debugging it with code changes to the test.